src/logic/WordsAndCharacters.java

```diff
@@ -0,0 +1,59 @@
+package logic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.TreeSet;
+
+public abstract class WordsAndCharacters {
+    public static final String[] POLISH_ALPHABET = {"a", "ą", "b", "c", "ć", "d", "e", "ę",
+                                                    "f", "g", "h", "i", "j", "k", "l", "ł",
+                                                    "m", "n", "ń", "o", "ó", "p", "r", "s",
+                                                    "ś", "t", "u", "w", "y", "z", "ź", "ż"};
+    public static void countCharactersAndShowDiagram(String content){
```

Comment:
biorac pod uwage ze uzyles tablic to jest zrobione dobrze. Ale lepsza struktura danych do tego rodzaju zliczen bylaby Mapa